lib/chef/resource/powershell_package_source.rb

```diff
@@ -78,28 +77,28 @@ class PowershellPackageSource < Chef::Resource
           if package_source_exists?
             converge_if_changed :url, :trusted, :publish_location, :script_source_location, :script_publish_location do
               update_cmd = build_ps_repository_command("Set", new_resource)
-              res = powershell_out(update_cmd)
-              raise "Failed to update #{new_resource.source_name}: #{res.stderr}" unless res.stderr.empty?
+              res = powershell_exec(update_cmd)
+              raise "Failed to update #{new_resource.source_name}: #{res.errors}" if res.error?
```

Comment:
Do we need to join the errors in these?

Reply:
I don't think so. The vast majority will have only one error. In the cases where there are multiple errors, I think having them comma separated in brackets makes it clear to view.